System-Testing-Deployments-and-Upgrades/Managing-User-Authorization/index.rst

```diff
@@ -0,0 +1,24 @@
+.. |author| replace:: *Tiago Ribeiro*
+.. |contributors| replace:: *none*
+
+.. _Managing-User-Authorization:
+
+###########################
+Managing User Authorization
+###########################
+
+.. note::
+
+    Managing user authorization through LOVE is still not fully implemented and tested.
+    This page will be updated once that is available.
```

Comment:
Consider:
The ability to manage user authentication using LOVE is not yet fully implemented nor tested.
This page will be updated once the functionality becomes available.

Reply:
In the first phrase I used the original start but with your ending.